docs/source/features/quantization/online_rotations.md

```diff
@@ -0,0 +1,189 @@
+# Introduction
```

Comment:
Add title `# AMD QUARK Online Rotation (experimental)`, adjust the header level accordingly.

Add `(quark_online_rotation)=` to the first line of this file.

Add `quark_online_rotation` to [here](https://github.com/vllm-project/vllm/blob/05e1fbfc52ca575e6539de63dbb5fab929683162/docs/source/features/quantization/index.md?plain=1#L20), after the line `quark`.